islandora_web_archive.install

```diff
@@ -28,3 +28,15 @@ function islandora_web_archive_uninstall() {
   variable_del('islandora_web_archive_make_warc_locally');
   islandora_install_solution_pack('islandora_web_archive', 'uninstall');
 }
+
+/**
+ * Print and log a message about possible lost physicalDescription data.
+ */
+function islandora_web_archive_update_7100(&$sandbox) {
+  drupal_set_message(filter_xss(t('The origin default form had incorrect XPath for the titleInfo block for relatedItem type="host" block. Existing mods:titleInfo fields will exist at the top level instead of nested in the relatedItem type="host" block. Please consult the !url for further information.',
```

Comment:
Shouldn't this just be the return value? https://api.drupal.org/api/drupal/modules!system!system.api.php/function/hook_update_N/7